### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug

/kind cleanup

> /kind documentation
> /kind feature
> /kind hotfix
> /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by markmandel — APPROVED** [review]

**Review by markmandel — APPROVED** [review]

**Comment by Kalaiselvi84** [comment]
I will follow the same approach for the helm commands that you mentioned in the other PR to create a .sh file for the Rust SDK.

**Comment by Kalaiselvi84** [comment]
@markmandel, I've included a shell script and a target to run the necessary `helm` commands for Agones update after you approve this PR. Also modified the release process based on the change. Could you please review the latest changes?

File: build/Makefile [hunk 1]
```diff
@@ -856,3 +856,10 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+update-agones: $(ensure-build-image)
```
Comment (markmandel) [inline build/Makefile:859]:
```suggestion
install-release: $(ensure-build-image)
```
Maybe, as a better name?

File: build/Makefile [hunk 2]
```diff
@@ -856,3 +856,10 @@ test-gen-crd-client:
...
+	docker run -it --rm \
+		$(common_mounts) \
+		-w $(workdir_path) \
+		-e "KUBECONFIG=/root/.kube/$(kubeconfig_file)" \
```
Comment (markmandel) [inline build/Makefile:863]:
```suggestion
```

Is this line required? I feel like it should be where it looks by default 🤔

File: build/Makefile [hunk 3]
```diff
@@ -856,3 +856,10 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+update-agones: $(ensure-build-image)
+	docker run -it --rm \
+		$(common_mounts) \
+		-w $(workdir_path) \
```
Comment (markmandel) [inline build/Makefile:862]:
```suggestion
```

Given that this is all helm commands - probably doesn't need to be in a specific work dir?

**Review by markmandel — COMMENTED** [review]
Looks good, just some ideas on naming and some possible cleanup 👍🏻

File: build/Makefile (hunk 1 above)
Comment (Kalaiselvi84) [reply build/Makefile]: modified here and other locations as well

File: build/Makefile (hunk 2 above)
Comment (Kalaiselvi84) [reply build/Makefile]: excluded this line

File: build/Makefile (hunk 3 above)
Comment (Kalaiselvi84) [reply build/Makefile]: removed this line

**Review by markmandel — APPROVED** [review]
Looks good! Need to move it out of `draft` though please 😄

File: build/Makefile [hunk 4]
```diff
@@ -856,3 +856,8 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+install-release: $(ensure-build-image)
+	docker run -it --rm \
+		$(common_mounts) \
+		$(DOCKER_RUN_ARGS) \
+		$(build_tag) /go/src/agones.dev/agones/build/install-release.sh
```
Comment (aimuz) [inline build/Makefile:863]: Can you add a new blank line here?

**Pull request merged**